CommandLine/libAssets/assets.h

```diff
@@ -0,0 +1,23 @@
+/** Copyright (C) 2018 Robert B. Colton

```

Comment:
Rofl, I don't think that's how copyright works Josh, it's code one way or another, it's not like we're filing a patent on it lol. Also, we have a copyright notice in a lot less copyrightable areas of the source code already. Regardless, that doesn't change the fact I should have included you on the copyright since I am adapting your old code.